tutorials/zero-to-lamp-deploy-with-chef/index.md

```diff
@@ -0,0 +1,432 @@
+---
+title: Zero-to LAMP deploy on GCP with Chef
+description: Learn how to deploy a LAMP stack on GCP with Chef cookbooks.
+author: slevenick
+tags: Compute Engine, Cloud SQL, Chef
+date_published: 2018-10-16
+---
+
+## Objectives
+* Create a Chef Workstation in Google Cloud.
+* Use the Workstation to create a Google Compute Engine VM, Cloud SQL instance, firewall rules, and a database via Chef.
+* Deploy Apache and PHP on the VM to serve a simple page backed by the database instance.
+
+## Costs
+
+This tutorial uses billable components of Cloud Platform, including:
+
+* [Google Compute Engine][gce]
+* [Cloud SQL][csql]
+
+[gce]: https://cloud.google.com/compute/
+[csql]: https://cloud.google.com/sql/
+
+Use the [Pricing Calculator][pricing] to generate a cost estimate based on your
+projected usage.
+
+[pricing]: https://cloud.google.com/products/calculator
+
+## Before you begin
+
+1. Select or create a [Google Cloud Platform Console][console] project.
+[Go to the projects page][projects].
+1. Enable billing for your project. [Enable billing][billing].
+1. Install the [Google Cloud SDK][sdk].
+1. Authenticate `gcloud` with Google Cloud Platform.
+
+        gcloud init
+
+[console]: https://console.cloud.google.com/
+[projects]: https://console.cloud.google.com/project
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+[sdk]: https://cloud.google.com/sdk/
+
+## You will be executing commands on both a local machine (assuming Mac or Linux-based) and a Google Compute Engine VM. Command blocks will be prefixed with the bash comment (#) to indicate the correct machine.
```

Comment:
This should not be a heading (##).
```suggestion
You will be executing commands on both a local machine (assuming Mac or Linux-based) and a Google Compute Engine VM. Command blocks will be prefixed with the bash comment (#) to indicate the correct machine.
```

Reply:
I made this change during the production review.